src/store/createStore.ts

```diff
@@ -51,6 +119,10 @@ export const createStore = (options: CreateStoreOptions) =>
           return {
             network: true,
```

Comment:
`exploreEndpoints` below got me thinking. What do you think of returning `network: { id: true}` because currently our params has all the network information. For `TESTNET`, `FUTURENET`, and `MAINNET`, we do not need all the information since they aren't configurable. For example, `/?||=network$id=testnet&label=Testnet&horizonUrl=https:////horizon-testnet.stellar.org&rpcUrl=https:////soroban-testnet.stellar.org&passphrase=Test%20SDF%20Network%20/;%20September%202015;;`

It might be useful for `Custom`, but I also wonder if we can save that info in `localStorage`.